src/osd/OSDMap.cc

```diff
@@ -1022,16 +1022,16 @@ int OSDMap::calc_num_osds()
   return num_osd;
 }
 
-void OSDMap::count_full_nearfull_osds(int *full, int *nearfull) const
+void OSDMap::count_full_nearfull_osds(int& full, int& nearfull) const
```

Comment:
just a convention we are following, so it's clear that it's an output parameter at seeing a pointer parameter. and could you enlighten as to the advantage of passing by reference?